ontology/yaml/resources/states/states.yaml

```diff
@@ -25,6 +25,8 @@ OCCUPIED: Occupied sensor state or operation mode.
 UNOCCUPIED: Unoccupied sensor state or operation mode.
 VFD: Discrete speed (frequency) control
 BYPASS: Bypassing an element or entity.
+DRY: Modes of operation in DX unit
+FAN: Modes of operation in DX unit
```

Comment:
insufficient definition; elaborate and clarify. what does it actually mean for the device to be in FAN mode?

Reply:
The air-conditioner in FAN mode runs only the fan and heating/cooling is disabled.

Reply:
something like FAN_ONLY would be more helpful in making that clear.

Reply:
as well as that definition. please add it.